### ISSUE TITLE
Compile for Mac/Linux/Windows automatically with releases

### ISSUE THREAD

**Issue by IFcoltransG** [bladecoder/blade-ink-rs#10]
Ink-Proof's [dependency install script](https://github.com/chromy/ink-proof/blob/master/install_deps.py) downloads pre-compiled versions of each runtime for each operating system. To integrate into the official version of Ink Proof, Blade Ink would probably need to make executable versions of itself available online to download, across Linux, Mac and Windows. I'm guessing there'd be a way to do this with a github action every release, but I haven't looked into it.

**Comment by bladecoder** [comment]
Yes, I will create ASAP a GHA to automatically upload the package to crates.io and upload the binary `binkplayer` for differrent os to the release.

**Comment by bladecoder** [comment]
Created GHA to compile and upload binkplayer to the release, the action also uploads the packages to crates.io.

The release v1.0.1 has been created with the executables for win, mac and linux.

**Comment by IFcoltransG** [comment]
I've sent a PR to Ink Proof for v1.0.1.
